src/system/SearchModule/Resources/views/User/opensearch.xml.twig

```diff
@@ -7,7 +7,7 @@
     {% set q = '{searchTerms}' %}
     {% set page = '{startPage?}' %}
     {% set url = url('zikulasearchmodule_user_form', { q: 'QUERY', page: '999' }) %}
-    <Url type="text/html" template="{{ url|e('html_attr')|replace({'QUERY': q, '999': page }) }}"/>
+    <Url type="text/html" template="{{ url|replace({'QUERY': q, '999': page }) }}"/>
```

Comment:
also not a session fix but noticed problems in the output of this file. ping @cmfcmf